examples/dishwasher-app/silabs/data_model/dishwasher-thread-app.zap

```diff
@@ -3515,6 +3515,2370 @@
     },
     {
       "id": 2,
+      "name": "Anonymous Endpoint Type",
```

Comment:
We are adding 2000 lines here, some of which are basic clusters that would definitely be in the first 3000 lines, are we adding a new cluster? Otherwise, why are we doubling this zap file in size? Something seems off.